src/CoreWCF.ConfigurationManager/src/CoreWCF/Configuration/AddressHeaderCollectionElement.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Configuration;
+using System.Security;
+using System.Xml;
+using CoreWCF.Channels;
+
+namespace CoreWCF.Configuration
+{
+    public sealed class AddressHeaderCollectionElement : ServiceModelConfigurationElement
+    {
+        public AddressHeaderCollectionElement()
+        {
+        }
+
+        internal void Copy(AddressHeaderCollectionElement source)
+        {
+            if (source == null)
+            {
+                throw DiagnosticUtility.ExceptionUtility.ThrowHelperArgumentNull("source");
+            }
+
+            PropertyInformationCollection properties = source.ElementInformation.Properties;
+            if (properties[ConfigurationStrings.Headers].ValueOrigin != PropertyValueOrigin.Default)
+            {
+                this.Headers = source.Headers;
+            }
```

Comment:
Remove `this.` prefix. There is a Visual Studio intellicode helper to do this. Have the cursor on the `this` keyword and type ctrl-. and you should get the helper. There is an option to fix everywhere in the file.